tools/Performance/DynamoPerformanceTests/PerformanceTestConsoleApp.cs

```diff
@@ -118,19 +125,54 @@ private static void Compare(string baseResultsPath, string newResultsPath, strin
                 return;
             }
 
-            // Create Model comparer
-            Console.WriteLine("\nComparison of Model tests: \n");
-            var baseModelPath = Path.Combine(PerformanceTestHelper.GetFullPath(baseResultsPath), "DynamoPerformanceTests.DynamoModelPerformanceTestBase-report.csv");
-            var newModelPath = Path.Combine(PerformanceTestHelper.GetFullPath(newResultsPath), "DynamoPerformanceTests.DynamoModelPerformanceTestBase-report.csv");
-            var modelSavePath = Path.Combine(PerformanceTestHelper.GetFullPath(savePath), "DynamoPerformanceTests.Comparison-Model.csv");
-            var modelComparer = new ResultsComparer(baseModelPath, newModelPath, modelSavePath);
-
-            // Create View comparer
-            Console.WriteLine("\nComparison of View tests: \n");
-            var baseViewPath = Path.Combine(PerformanceTestHelper.GetFullPath(baseResultsPath), "DynamoPerformanceTests.DynamoViewPerformanceTestBase-report.csv");
-            var newViewPath = Path.Combine(PerformanceTestHelper.GetFullPath(newResultsPath), "DynamoPerformanceTests.DynamoViewPerformanceTestBase-report.csv");
-            var viewSavePath = Path.Combine(PerformanceTestHelper.GetFullPath(savePath), "DynamoPerformanceTests.Comparison-View.csv");
-            var viewComparer = new ResultsComparer(baseViewPath, newViewPath, viewSavePath);
+            ResultsComparer modelComparer = null;
+            ResultsComparer viewComparer = null;
+
+            try
+            {
+
+                // Create Model comparer
+                Console.WriteLine("\nComparison of Model tests: \n");
+                var baseModelPath = Path.Combine(PerformanceTestHelper.GetFullPath(baseResultsPath), "DynamoPerformanceTests.DynamoModelPerformanceTestBase-report.csv");
+                var newModelPath = Path.Combine(PerformanceTestHelper.GetFullPath(newResultsPath), "DynamoPerformanceTests.DynamoModelPerformanceTestBase-report.csv");
+                var modelSavePath = Path.Combine(PerformanceTestHelper.GetFullPath(savePath), "DynamoPerformanceTests.Comparison-Model.csv");
```

Comment:
Maybe out of scope but seems we better define these as static strings or even put to resources in case we will localize these in the future?

Reply:
yeah, can do that, I don't think we'll be localizing this though.